src/app/services/auth.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { User } from '../models/user';
+import { Observable, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  loggedIn: boolean = false;
+
+  logIn(user: User): Observable<boolean> {
+    if (user.username === 'admin' && user.password === 'admin') {
+      this.loggedIn = true;
+    }
```

Comment:
For consistency with mocking what would be an api endpoint, add an else statement to throw an error if the credentials don't match. Essentially mimicking the flow of a `403 - Unauthorized` http error response.